# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: ipc/glue/UtilityProcessChild.h
# Commit: cecf09ccbda3
# Full Hash: cecf09ccbda3f2ba21a49b13e6d8c83f61989cee
# Author: Ray Kraesig <rkraesig@mozilla.com>
# Date: 2023-11-08 21:12:03
# Description:
#   Bug 1863044 - remove unneeded variable and associated MOZ_ASSERT  r=ipc-reviewers,nika
#   
#   `mFileDialogInstance` is not needed, since we don't reuse old file-
#   dialog actors. (We _do_ reuse active file-dialog utility processes as
#   hosts for those actors; this check is a leftover from when that was
# ==============================================================================

diff -r b47bf0a3b904 -r cecf09ccbda3 ipc/glue/UtilityProcessChild.h
--- a/ipc/glue/UtilityProcessChild.h	Wed Nov 08 16:56:44 2023 +0000
+++ b/ipc/glue/UtilityProcessChild.h	Wed Nov 08 17:25:57 2023 +0000
@@ -100,7 +100,6 @@
   RefPtr<dom::JSOracleChild> mJSOracleInstance{};
 #ifdef XP_WIN
   RefPtr<PWindowsUtilsChild> mWindowsUtilsInstance;
-  RefPtr<PWinFileDialogChild> mFileDialogInstance;
 #endif
 
   AsyncBlockers mShutdownBlockers;
